docs/tutorials/1_the_ibm_quantum_account.ipynb

```diff
@@ -430,7 +430,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The `qiskit-ibm`'s implementation of `BaseBackend` is [`IBMBackend`](https://qiskit.org/documentation/stubs/qiskit_ibm.IBMBackend.html#qiskit_ibm.IBMBackend). It accepts additional parameters to the `run()` method:\n",
+    "The `qiskit-ibm`'s implementation of `BaseBackend` is [`IBMBackend`](https://qiskit.org/documentation/stubs/qiskit_ibm_provider.IBMBackend.html#qiskit_ibm_provider.IBMBackend). It accepts additional parameters to the `run()` method:\n",
```

Comment:
Yes, these inks will not work. But pointing to qiskit_ibm is also confusing. So decided to leave it like this for now...